File: EVAL.cpp
#include <iostream>
using namespace std;
int main(){
	long long denom=19,num=17,n=17,temp;
	while(true){
		temp=denom;
		denom=n*denom+num;
		num=(--n)*temp;

		if(n==0){num=temp*10;break;}
	}//cout <<num <<"  "<<denom <<endl;
	cout <<"2.";
    for(int i=0;i<1000000;i++){
        cout<<num/denom;
        num=(num%denom)*10;
    }
   return 0;
}

File: EVAL.py
import sys
denom,num,n=16002,16000,16000
while True:
	temp=denom;
	denom=n*denom+num
	n-=1
	num=n*temp;
	#print 'jksk ',
	if n==0:
		num=temp*10
		break
print '2.',;sys.stdout.softspace=False;
for i in range(80000):
	print num/denom,;sys.stdout.softspace=False;
	num=(num%denom)*10;